src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -80,14 +81,17 @@
      * Opens the given Inventory for the player to view.
      *
      * @param inventory The inventory to view
+     * @param cause The {@link Cause} to use when opening the inventory
      */
-    void openInventory(Inventory inventory);
+    void openInventory(Inventory inventory, Cause cause);
```

Comment:
@Zidane: I believe we discussed this a while back, as requiring/allowing a `Cause` parameter for all methods which fire an event. Thoughts?